### ISSUE TITLE
raft blog post

### ISSUE THREAD

**Issue by raphaelauv** [Snapchat/KeyDB#482]
following the blog post https://docs.keydb.dev/blog/2021/07/06/blog-post

could you summarize the advantage of using RAFT for a redis ( keydb ) cluster against the current cluster redis protocol ?

thanks

**Comment by msotheeswaran** [comment]
RAFT provides stricter consistency as it requires a majority of the cluster to acknowledge a change before committing it whereas the current cluster protocol will commit the change immediately then notify the rest of the cluster afterwards.

**Comment by raphaelauv** [comment]
Thanks
